src/mots/config.py

```diff
@@ -200,8 +201,25 @@ def clean(file_config: FileConfig, write: bool = True):
 
     bmo_data = get_bmo_data(people)
     updated_people = People(people, bmo_data)
-    logger.debug("Updating people configuration based on BMO data...")
-    file_config.config["people"] = updated_people.serialized
+
+    for person in people:
+        if "nick" not in person:
+            person["sync"] = True
+
+    if refresh:
+        # Use the updated list that was synchronized with Bugzilla.
+        logger.info("Refreshing all people entries from Bugzilla.")
+        file_config.config["people"] = updated_people.serialized
+    else:
+        # Use the original people list, and update entries only where needed.
+        logger.warning("Only synchronizing new people with Bugzilla.")
+        updated_people_dict = {p["bmo_id"]: p for p in updated_people.serialized}
```

Comment:
`p` -> `updated_person`

Reply:
I personally don't think this does much in terms of increasing readability or understanding of the dictionary comprehension. Not saying that it makes things worse, but it is implied that `updated_people` is a list containing people and the scope of `p` is in this single line, this variable is not used anywhere outside this line. This applies to most of the dictionary comprehensions in this PR.

```suggestion
        updated_people_dict = {
            updated_person["bmo_id"]: updated_person
            for updated_person in updated_people.serialized
        }   
```